mondrian/src/main/java/mondrian/rolap/aggmatcher/ExplicitRules.java

```diff
@@ -1209,7 +1210,7 @@ protected Recognizer.Matcher getMeasureFactCountMatcher() {
                 public boolean matches(String name) {
                     HashSet<Column> measuresFactCountSet =
                             new HashSet<>(measuresFactCount.values());
-                    return measuresFactCountSet.contains(name);
+                    return measuresFactCountSet.stream().anyMatch(c -> name.equals(c.getName()));
```

Comment:
Maybe fist use
.map(Column::getName).anyMatch


I not sure please test

Reply:
And also may .filter( objects:notnull)

Reply:
agree

Reply:
done